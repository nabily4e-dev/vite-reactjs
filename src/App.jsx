import './App.css'
import HelloWorld from './components/HelloWorld/HelloWorld.jsx'

function App() {
	return (
		<main>
			<HelloWorld msg='Welcome ^^' />
			<h1>Vite + React</h1>
		</main>
	)
}

export default App
