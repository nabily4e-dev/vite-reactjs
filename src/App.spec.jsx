import { render } from '@testing-library/react'
import { expect, it } from 'vitest'

import App from './App.jsx'

it('renders <App> properly', () => {
	const { asFragment } = render(<App />)

	expect(asFragment()).toMatchSnapshot()
})
