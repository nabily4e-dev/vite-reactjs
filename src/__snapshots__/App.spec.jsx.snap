// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`renders correctly 1`] = `
<DocumentFragment>
  <main>
    <div
      class="_HelloWorld_ea6a9b"
    >
      <h1
        data-testid="title"
      >
        Welcome ^^
      </h1>
      <p>
        Recommended IDE setup: 
        <a
          href="https://code.visualstudio.com/"
          rel="noreferrer"
          target="_blank"
        >
          VSCode
        </a>
      </p>
      <p>
        See 
        <code>
          README.md
        </code>
         for more information.
      </p>
      <p>
        <a
          href="https://vitejs.dev/guide/features.html"
          rel="noreferrer"
          target="_blank"
        >
          Vite Docs
        </a>
         | 
        <a
          href="https://reactjs.org/"
          rel="noreferrer"
          target="_blank"
        >
          React Docs
        </a>
      </p>
      <button
        type="button"
      >
        count is: 0
      </button>
      <p>
        Edit 
        <code>
          components/HelloWorld/HelloWorld.jsx
        </code>
         to test hot module replacement.
      </p>
    </div>
    <h1>
      Vite + React
    </h1>
  </main>
</DocumentFragment>
`;
